packages/web-api/src/WebClient.spec.js

```diff
@@ -115,6 +115,46 @@ describe('WebClient', function () {
     });
   });
 
+  describe('has an option to override the Axios timeout value', function () {
+    it('should log warning and throw error if timeout exceeded', function (done) {
+      const timeoutOverride = 15; // ms, guaranteed failure
```

Comment:
If there is no reason to choose 15 milliseconds here, making it as small as possible (say, one millisecond, I don't know it works, though) may be safer. Perhaps, 15 also works in your environment and may work with the current builds. But I have been observing that GitHub Actions runtime sometimes behaves differently.

Reply:
You were right -- I wouldn't have thought that it would make a difference but reducing it to 1ms eliminating the failing tests. Thanks for sharing this insight!